### PR TITLE
Add .config folder to .gitignore

### PR DESCRIPTION
**Pull request by selectthegang**
Read the title >3

### CODE REVIEW

**Review by mat-1 — DISMISSED** [review]
Why are there Python things in the gitignore? This PR doesnt seem necessary, Repldex works fine in Replit. Also please look at the diff.

**Review by Codemonkey51 — CHANGES_REQUESTED** [review]
You removed node_modules as well as other necessary parts of the gitignore…

**Comment by selectthegang** [comment]
Weird, i cloned the repl and added .config and ./config.
I also ran npm i which is why there are more changes
the pr is necessary because when you clone the repo, it automatically creates a .config folder for Nix. This makes that folder not hit GitHub commits

**Comment by selectthegang** [comment]
@mat-1 @Codemonkey51 fixed

**Comment by selectthegang** [comment]
should be okay now

File: .gitignore [hunk 1]
```diff
@@ -8,4 +8,7 @@
 /.vercel_build_output
 .vercel
 /.cache
-/.upm
\ No newline at end of file
+/.upm
+./.config
```
Comment (Codemonkey51) [inline .gitignore:12]: you only need the `.config` not `./.config` or `/.config`

**Review by Codemonkey51 — DISMISSED** [review]

File: .gitignore (hunk 1 above)
Comment (selectthegang) [reply .gitignore:12]: Ight, i will make fix commit

Comment (selectthegang) [reply .gitignore]: pushed

**Review by mat-1 — APPROVED** [review]

**Pull request merged**